opus/application/apps/dictionary/secrets_template.py

```diff
@@ -22,9 +22,6 @@
 # For PostgreSQL this is the schema name.
 DICTIONARY_SCHEMA_NAME = 'XXX'
 
-# The directory where the pds-tools repo lives.
-PDS_TOOLS_PATH = 'XXX'
-
```

Comment:
Just would like to point out that`PDS_TOOLS_PATH` is removed here, but it's still at line 29-30 in `pds-opus/lib/opus_support.py`